include/dronecore.h

```diff
@@ -30,17 +59,83 @@ class DroneCore
         CONNECTIONS_EXHAUSTED
     };
 
+    /**
+     * Translates the ConnectionResult enum to a human-readable English string.
+     */
     static const char *connection_result_str(ConnectionResult);
 
+    /**
+     * Adds a UDP connection with the default arguments.
+     *
+     * This will listen on UDP port 14540 on localhost.
+     *
+     * @return the result of adding the connection.
+     */
     ConnectionResult add_udp_connection();
+
+    /**
+     * Adds a UDP connection with a specific port number.
```

Comment:
Always on localhost? Worth saying here too? 
Would you ever want to listen to some other UDP address than localhost?

Reply:
This is the local port that it listens on, so I don't think it can be anything else than localhost. The packet can be sent from any remote IP though.

It's probably best to avoid mentioning localhost to avoid confusion.